library/src/main/java/com/rockerhieu/emojicon/EmojiconGridFragment.java

```diff
@@ -66,8 +67,7 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
             mData = People.DATA;
             mUseSystemDefault = false;
         } else {
-            Object[] o = (Object[]) getArguments().getSerializable("emojicons");
-            mData = Arrays.asList(o).toArray(new Emojicon[o.length]);
+            mData = (Emojicon[]) getArguments().getParcelableArray(EMOJICONS_KEY);
```

Comment:
@uwemaurer Maybe I am missing something but I can't reproduce this on emulators API 19 and 23. Anyway, I've sent a pull request (https://github.com/rockerhieu/emojicon/pull/79).